src/Driver/WpcliDriver.php

```diff
@@ -111,25 +123,19 @@ public function wpcli($command, $subcommand, $raw_arguments = [])
             $config = sprintf('--path=%s --url=%s', escapeshellarg($this->path), escapeshellarg($this->url));
         }
 
-        // Support Windows.
-        if (DIRECTORY_SEPARATOR === '\\') {
-            $binary = 'wp.bat';
-        } else {
-            $binary = 'wp';
-        }
-
-        // Adjust bootstrap path, dependant on if WordHat is a Composer dependancy, or being run directly.
-        $package = '/vendor/paulgibbs/behat-wordpress-extension';
-        if ($package === substr(getcwd(), -strlen($package), strlen($package))) {
-            $package = '';
-        }
+        //Absolute path to the WordHat src directory
+        $src = dirname( __DIR__ );
 
         $cmd_output = [];
         $exit_code  = 0;
-        $wpcli_args = sprintf('--no-color --require=%1$s%2$s/src/WpcliLogger.php', getcwd(), $package);
+        $wpcli_args = sprintf('--no-color --require=%1$s/WpcliLogger.php', $src);
 
         // Query WP-CLI.
-        exec("{$binary} {$config} {$wpcli_args} {$command} {$subcommand} {$arguments} 2>&1", $cmd_output, $exit_code);
+        exec(
+         "{$this->binary} {$config} {$wpcli_args} {$command} {$subcommand} {$arguments} 2>&1",
```

Comment:
What should the indentation be? (Again I only put this on a new line to satisfy `phpcs` - and assumed it would yell at me if the indentation was wrong ;). We are using spaces not tabs, right?).